test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1126,6 +1131,13 @@ def _revert_pbsconf_server(self, vals_to_set):
                 new_pbsconf["PBS_LOG_HIGHRES_TIMESTAMP"] = "1"
                 restart_pbs = True
 
+            # if shasta, set PBS_PUBLIC_HOST_NAME
+            if server.platform == 'shasta':
+                localhost = socket.gethostname()
+                if new_pbsconf["PBS_PUBLIC_HOST_NAME"] != localhost:
+                    new_pbsconf["PBS_PUBLIC_HOST_NAME"] = localhost
+                    restart_pbs = True
```

Comment:
Isn't PBS_PUBLIC_HOST_NAME there for the commands?  If it is, we won't need to restart PBS.

Reply:
In order to set the pbsconfig, restart_pbs needs to be true.